docs/UserGuide.md

```diff
@@ -158,19 +158,26 @@ These are all the features in regard to a member in CMC.
 Adds a member to the list of members in CMC.
 
 - Use `addMember` or `addm` to use add a member
-- Use `addApplicant` or `adda` to use add an applicant
 
 #### Usage:
 
-##### Adding member:
-
 `addMember /name {memberName} /phone {phoneNumber} /email {email} /tele {telegramHandle} /tag {tag}`
 
 `addm /name {memberName} /phone {phoneNumber} /email {email} /tele {telegramHandle} /tag {tag}`
 
 #### Acceptable values for parameters:
 
-- `name`: Only alphabetical characters, @, () are allowed
+<div markdown="span" class="alert alert-primary">
+:information_source: **Notes about duplicate members:**
+
+Duplicate members are not allowed to be added into the list of members.
+
+An member is considered a duplicate if the member's name, phone number, email **or** telegram handle matches that of an existing member.
+
+This means that only a member with a unique name, phone number, email and telegram handle can be added into the list of members.
+</div>
+
+- `memberName`: Only alphabetical characters, @, () are allowed
```

Comment:
![image](https://github.com/AY2324S1-CS2103T-W15-3/tp/assets/120372506/4f9f51f7-a3f5-4ece-88ae-5f4a0ff4dd7f)

I think we wanted input parameters to be in callout boxes like in the picture above! Also might need to change the email format to match the actual regex validation of the code, as well as add this for tag and task

Reply:
Added issue for alfred to edit that